doc/contrib/frontend.md

```diff
@@ -78,6 +78,32 @@ Take of example a button component that is used as the login button.
 
 That has the added benefit, that presentational components can be easily integrated into Storybook.
 
+## Documentation
+
+We use Storybook [Storybook](https://storybook.js.org) to document shared components.
+Story files are named `*.stories.tsx` and should be located in the same folder as the component they are documenting.
+For an example please take a look in the `src/components` folder.
+
+For other API documentation like Hooks and utility functions, we use [TypeDoc](https://typedoc.org/).
+To generate the TypeDoc documentation under `src/generated/docs` simply run `npm run typedoc`.
+The generated documentation is automatically integrated into Storybook under the menu `DOCS`.
+To display an arbitrary other `.mdx` file in the `DOCS` menu, it has to export a header like this.
+The `.mdx` files should have valid [MDX2](https://mdxjs.com/blog/v2/) syntax, otherwise Storybook will crash.
```

Comment:
Is there some syntax checker that doesn't crash?

Reply:
I guess we will not be writing much manual mdx by ourselves, but there are not many rules, so I think automatic syntax checking, just for that, is too much effort. Maybe there are plugins of the IDEs but I don't know.

Reply:
@markus2330 Sorry, also not crash, it will display an error on viewing the said story.

Reply:
Ok, then forget my comment and simply fix the text :wink: